plugin/adsk/scripts/mayaUsd_createStagesFromExistingLayer.mel

```diff
@@ -13,17 +13,28 @@
 // limitations under the License.
 //
 
-global string $stageFromExistingLayer_primPath = "";
-global string $stageFromExistingLayer_excludePrimPath = "";
-global int $stageFromExistingLayer_loadPayloads = 1;
+proc setOptionVars (int $forceFactorySettings)
```

Comment:
Changing all of the global variables to optionVar's so they persist.

Reply:
This proc can be called either to init the variables or reset them depending on the forceFactorySettings param.